packages/terra-form-select/src/tag/Frame.jsx

```diff
@@ -450,6 +444,20 @@ class Frame extends React.Component {
     }
   }
 
+  /**
+   * Handles the mouse down events.
+   * @param {event} event - The mouse down event.
+   */
+  handleMouseDown(event) {
+    // Preventing default events stops the search input from losing focus.
+    // The default variant has no search input therefore the mouse down gives the component focus.
+    event.preventDefault();
+    if (event.target.hasAttribute('data-terra-form-select-input')) {
+      this.input.focus();
+    }
+    this.openDropdown(event);
+  }
```

Comment:
Same comment mentioned above for the multiple variant, this function is never called

Reply:
Removed